docs/rst/user_manual/wan_configuration.rst

```diff
@@ -68,13 +68,86 @@ These are a list of tips to help choosing whether to use one or the other.
         - Require only server side of the communication |br|
           to have port forwarded from the router.
 
-.. note:
+.. note::
 
     DDS is thought to work over UDP and has its own reliability mechanisms.
     Thus, the |ddsrouter| uses UDP transport by default for every address that has not explicitly specified
     a transport in the configuration file.
 
 
+TLS
+---
+
+|eddsrouter| also supports `TLS over TCP <https://fast-dds.docs.eprosima.com/en/latest/fastdds/transport/tcp/tls.html>`_,
+and its configuration can be set per participant for types Local Discovery Server and WAN. Following is a list of the
+accepted entries under the ``tls`` tag:
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Tag
+        - Requiredness
```

Comment:
This word seems not common (and is not included in sphinx dictionary). I will use other, maybe `Required`

Reply:
Not very common, probably old-fashioned or nonexistent, but for the Computer Science domain should be ok I think.